geopandas/tools/geocoding.py

```diff
@@ -11,15 +11,17 @@
 
 
 def _throttle_time(provider):
-    """ Amount of time to wait between requests to a geocoding API.
-
-    Currently implemented for GeocodeFarm, as their terms of service
-    require a maximum of 1 request per second.
-    https://wiki.openstreetmap.org/wiki/GeocodeFarm_usage_policy
+    """
+    Amount of time to wait between requests to a geocoding API, for providers
+    that specify rate limits in their terms of service.
     """
     import geopy.geocoders
-    if provider == geopy.geocoders.GeocodeFarm:
+    # https://wiki.openstreetmap.org/wiki/GeocodeFarm_usage_policy
+    if provider == geopy.geocoders.Nominatim:
         return 1
+    # https://geocode.farm/geocoding/free-api-documentation/
+    elif provider == geopy.geocoders.GeocodeFarm:
+        return 0.25
```

Comment:
I am not very familiar with those code, but: doesn't this make it impossible to use GeocodeFarm with an API key? (where you would have a higher number of requests per second available, and you don't want to go so slow)

Reply:
You're absolutely right, I wasn't thinking of that, but I removed it.

I don't see a good way to use `RateLimiter` from `geopy` without a major revision to how we're handling geocoding in `geopandas`, which is more than I'm up for at present. Do you have any opinions on how to proceed?